app/assets/stylesheets/forever_style_guide/modules/_modal.scss

```diff
@@ -4,6 +4,8 @@ $modal-product-suggestion-sm: 170px;
 $modal-promo-height: 475px;
 $modal-xl-screen-md-width: 900px;
 $modal-xl-screen-lg-width: 1170px;
+$modal-margin-top-sm: 45px;
```

Comment:
I think i'd expect this to be 32px, which is defined elsewhere as $ambassador-banner-text-height

![screen shot 2018-04-05 at 5 24 38 pm](https://user-images.githubusercontent.com/3814998/38392654-4882c01a-38f6-11e8-863b-3897171634e3.png)